src/components/pages/dashboard/panels/map/azureMap.js

```diff
@@ -31,11 +41,22 @@ export class AzureMap extends Component {
     return false;
   }
 
-  initializeMap(azureMapsKey) {
+  initializeMap(theme, azureMapsKey) {
     this.map = new AzureMaps.Map('map', {
       'subscription-key': azureMapsKey,
       center: Config.mapCenterPosition,
-      zoom: 11
+      zoom: 11,
+      style: this.getMapStyle(theme)
+    });
+
+    var styleSelector = new AzureMaps.control.StyleControl();
```

Comment:
const

Reply:
Darn it! That's what I get for cut&Paste sample code!  Haha.